lib/reverse_words.rb

```diff
@@ -1,2 +1,20 @@
+#To test my code in repl.it, created an str variable using the line below:
+#str = "Today is a good day".split(" ")
 def reverse_words(str)
+arr_end = str.length - 1
```

Comment:
Right now, for the test input `"Today is a good day"`, your method ends up with a value of `"y a d   d o o g   a   s i   y a d o T"` for `fnl_arr`, rather than `"day good a is Today"`. Unlike the first method, rather than iterating over the entire string passed into the method, what will you need to iterate through in order to "bin" each word as an in-tact string?